app/src/main/res/values-zh-rCN/strings.xml

```diff
@@ -36,10 +38,12 @@
     <string name="show_grid_pref">赤经赤纬网格</string>
     <string name="show_horizon_pref">水平线</string>
     <string name="show_sky_gradient">天空梯度</string>
+    <string name="show_meteors_pref">Meteor Showers</string>
```

Comment:
This shouldn't be needed - in the absence of a translation the app should already fall back to the English string.